Packet++/header/GvcpLayer.h

```diff
@@ -0,0 +1,776 @@
+#pragma once
+
+#include "IpAddress.h"
+#include "Layer.h"
+#include "MacAddress.h"
+#include "SystemUtils.h"
+#include <iostream>
+
+/**
+ * @file GvcpLayer.h
+ * @author An-Chi Liu (phy.tiger@gmail.com)
```

Comment:
nit: I'm not strictly against it, but we usually don't put the author inside the code.

ditto in the other new files in this PR

Reply:
Okay.